### PR TITLE
Seeding Postgres 

### PR DESCRIPTION
**Pull request by reedbruskin**
- dataGenerator script now uses pure Node to generate 10M records.
- Postgres seeding completed.

### CODE REVIEW

File: database/dataGenerator.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    let us_free_shipping = faker.random.boolean();
+    let carts_item_is_in = faker.random.number();
+
+    // create new value in CSV file
+    writer.write(`${item_name},${best_seller},${price},${price_reduction},${in_stock},${us_free_shipping},${carts_item_is_in}\n`)
+  }
+  writer.end();
+}
+
+shoppingDataGeneration(10_000_000);
```
Comment (7MIMIRA) [inline database/dataGenerator.js]: Great use of underscore as a numeric separator!

File: database/seedPostgres.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const db = require('./index.js');
+
+// Insert data into DB script below
+// TODO: separate into separate file and get working.
+const insertData = async () => {
+  let csvFile = __dirname + '/seed10M.csv';
+  let query = `COPY items (item_name, best_seller, price, price_reduction, in_stock, us_free_shipping, carts_item_is_in) FROM '${csvFile}' DELIMITER ',' CSV HEADER;`
+  db.query(query, (err, res) => {
+    if (err) {
+      console.log(err)
```
Comment (7MIMIRA) [inline database/seedPostgres.js]: Add a semicolon at the end of the line if you are worried about them. Not a big deal.

**Review by 7MIMIRA — APPROVED** [review]
Looking good!

File: database/seedPostgres.js (hunk 2 above)
Comment (reedbruskin) [reply database/seedPostgres.js]: Thanks - forgot to run eslint with this commit, but I'll do that to add semis.

**Pull request merged**